### PR TITLE
hostd integrity check sector data and general dataset refresh intervals

### PR DESCRIPTION
**Pull request by alexfreska**
- Integrity check alert sector data will now update when there is an ongoing integrity check.
- All data will now refresh more frequently.

### CODE REVIEW

**Comment by n8maninger** [comment]
Can everything refresh more frequently? Maybe every 15 seconds?

**Review by ChrisSchinnerl — DISMISSED** [review]

**Comment by alexfreska** [comment]
@n8maninger currently the change is to 20s when viewing alerts, 60s in background. Are you suggesting 15s when viewing?

**Comment by n8maninger** [comment]
Yeah. Im suggesting every data set should refresh at a 10-15s interval.

**Comment by alexfreska** [comment]
@n8maninger alright, updated all apps to use a default of 15s for all primary data.

File: apps/hostd/dialogs/AlertsDialog.tsx [hunk 1]
```diff
@@ -19,7 +20,15 @@ type Props = {
 }
 
 export function AlertsDialog({ open, onOpenChange }: Props) {
-  const alerts = useAlerts()
+  const alerts = useAlerts({
```
Comment (n8maninger) [inline apps/hostd/dialogs/AlertsDialog.tsx:23]: This should always use the default too imo

Comment (alexfreska) [reply apps/hostd/dialogs/AlertsDialog.tsx:23]: oop, missed that one, ty.

**Review by n8maninger — APPROVED** [review]

**Pull request merged**